src/action/method.ts

```diff
@@ -0,0 +1,61 @@
+import { Account } from "../account/account";
+import { IRpcMethod } from "../rpc-method/types";
+import { toAction, toActionTransfer } from "../rpc-method/types";
+import { Transfer } from "./types";
+
+export class TransferMethod {
+  public client: IRpcMethod;
+  public account: Account;
+  public transfer: Transfer;
+
+  constructor(client: IRpcMethod, account: Account, transfer: Transfer) {
+    this.client = client;
+    this.account = account;
+    this.transfer = transfer;
+  }
+
+  public async execute(): Promise<string> {
+    const tf = toActionTransfer({
+      amount: this.transfer.amount,
+      recipient: this.transfer.recipient,
+      payload: this.transfer.payload
+    });
+
+    const meta = await this.client.getAccount({
+      address: this.account.address
+    });
+
+    const action = toAction({
+      // @ts-ignore
```

Comment:
please do not ignore

Reply:
![Screen Shot 2019-04-09 at 11 22 02 PM](https://user-images.githubusercontent.com/1387285/55856145-4e298080-5b1e-11e9-8f89-ca20e8d7fe02.png)

Reply:
![Screen Shot 2019-04-09 at 11 22 39 PM](https://user-images.githubusercontent.com/1387285/55856183-63061400-5b1e-11e9-9b84-daec41d5cc99.png)

if the action does not match. we need to make it match.